src/lem/path.rs

```diff
@@ -64,129 +64,160 @@ impl Path {
+        for op in self.ops {
+            match op {
+                Op::Call(out, func, inp) => {
+                    let out = map.get_many_cloned(&out)?;
+                    let inp = map.get_many_cloned(&inp)?;
+                    // We will always assume previously defined `Func`s to already be deconflicted.
+                    // Since variables are only an issue in parallel paths, not sequential ones,
+                    // there's no need to deconflict `func`
+                    ops.push(Op::Call(out, func, inp))
+                }
+                Op::Null(ptr, tag) => ops.push(Op::Null(insert_one(map, path, &ptr), tag)),
+                Op::Hash2(img, tag, preimg) => {
+                    let preimg = map.get_many_cloned(&preimg)?.try_into().unwrap();
+                    let img = insert_one(map, path, &img);
+                    ops.push(Op::Hash2(img, tag, preimg))
+                }
+                Op::Hash3(img, tag, preimg) => {
+                    let preimg = map.get_many_cloned(&preimg)?.try_into().unwrap();
+                    let img = insert_one(map, path, &img);
+                    ops.push(Op::Hash3(img, tag, preimg))
+                }
+                Op::Hash4(img, tag, preimg) => {
+                    let preimg = map.get_many_cloned(&preimg)?.try_into().unwrap();
+                    let img = insert_one(map, path, &img);
+                    ops.push(Op::Hash4(img, tag, preimg))
+                }
+                Op::Unhash2(preimg, img) => {
+                    let img = map.get_cloned(&img)?;
+                    let preimg = insert_many(map, path, &preimg);
+                    ops.push(Op::Unhash2(preimg.try_into().unwrap(), img))
+                }
+                Op::Unhash3(preimg, img) => {
+                    let img = map.get_cloned(&img)?;
+                    let preimg = insert_many(map, path, &preimg);
+                    ops.push(Op::Unhash3(preimg.try_into().unwrap(), img))
+                }
+                Op::Unhash4(preimg, img) => {
+                    let img = map.get_cloned(&img)?;
+                    let preimg = insert_many(map, path, &preimg);
+                    ops.push(Op::Unhash4(preimg.try_into().unwrap(), img))
+                }
+                Op::Hide(..) => todo!(),
+                Op::Open(..) => todo!(),
+            }
+        }
+        let ctrl = self.ctrl.deconflict(path, map)?;
+        Ok(Block { ops, ctrl })
+    }
+
+    /// Computes the number of possible paths in a `Block`. `Func`s withen `Call`s are
+    /// treated as black boxes, therefore we do not count paths within a call.
+    fn num_paths(&self) -> usize {
+        match &self.ctrl {
+            Ctrl::MatchTag(_, cases) => {
+                cases.values().fold(0, |acc, block| acc + block.num_paths())
+            }
+            Ctrl::MatchSymbol(_, cases, _) => {
+                cases.values().fold(0, |acc, block| acc + block.num_paths())
+            }
```

Comment:
We also need to account for the paths in the default case of `MatchSymbol`

Reply:
Hmm this is in master too. I'll fix this in this branch